gradle-scripts/docker.gradle

```diff
@@ -1,7 +1,7 @@
 docker {
     javaApplication {
         baseImage = 'java:openjdk-8-jre-alpine'
-        maintainer = 'Hesham Massoud [hesham.massoud@commercetools.com]'
+        maintainer = 'PS Team Munich [ps-dev@commercetools.de]'
```

Comment:
Sad for Hesham 😅

Reply:
dot com is better.

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/136/commits/caec734b944426eedb4af0343ca86c89a32f5e1e